yaml/slides_prod/slides_ingress_prod.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    field.cattle.io/publicEndpoints: '[{"addresses":["10.100.1.8"],"port":443,"protocol":"HTTPS","serviceName":"pitc-ansible-training-port:ansible-slides","ingressName":"pitc-ansible-training-prod:ansible-slides","hostname":"ansible-slides.k8s.puzzle.ch","path";"/","allNodes":true}]'
```

Comment:
do we need those annotations and internal ip here?

Reply:
Don't think so. Remove it